Expensify/App
# [No QA] Fix Storybook to fix staging web deploys

### Details
Storybook was broken by https://github.com/Expensify/App/pull/9750. However, this caused staging web deploy to fail:

https://github.com/Expensify/App/runs/7243680102?check_suite_focus=true

So we needed to revert or fix storybook in order to fix staging deploys. I chose to fix it without reverting.

### Fixed Issues
$ n/a – broken deploys

### Tests
1. Run `npm i`
1. Run `npm run storybook`
1. Verify that storybook works locally in your browser.
1. Run `npm run storybook-build`
1. Verify that the build completes successfully.
1. Merge this PR
1. Verify that the staging deploy happens successfully.
